apps/pp/models.py

```diff
@@ -102,7 +102,16 @@ class Meta:
         app_label = 'pp'
         unique_together = [('user', 'reference')]
 
-    # No JSONAPIMeta information; the model corresponds to two resources, not just one
+    class JSONAPIMeta:
+        useful_resource_name = 'usefuls'
+        objection_resource_name = 'objections'
+
+        @classmethod
+        def get_resource_names(cls, obj=None):
+            return {
+                cls.useful_resource_name: getattr(obj, 'useful', False),
+                cls.objection_resource_name: getattr(obj, 'objection',  False),
+            }
```

Comment:
I'm wondering what's the reason behind "one-true" dict instead of the resource name. As far as I remember this representation is used somewhere internally. If that's the reason, I think we should convert it to the more straightforward output in the future.

Reply:
We needed not only current resource_name, but _ALL_ resource names, from which only one is not null at a time. We will have got rid of this objection/useful dualism in the future. In this case it was really relatively straightforward solution as this relation was completely not straightforward ;)